snappi/snappigenerator.py

```diff
@@ -701,9 +701,16 @@ def _get_property_param_string(self, yobject):
                     properties.append(name)
                     if 'default' in property:
                         default = property['default']
-                    if 'enum' in property or 'format' in property:
+                    if 'enum' in property:
                         val = "=%s" % default if default == 'None' else "='%s'" % default
                         property_param_string += val
+                    elif 'format' in property:
```

Comment:
This change is correct but shouldn't we have actually modified the model from type string to type integer ? (currently I see that bgpv4routeaddress params are defined with type string and format integer)